db/migrate/20191211174028_create_pictures.rb

```diff
@@ -1,11 +0,0 @@
-# frozen_string_literal: true
-
-class CreatePictures < ActiveRecord::Migration[6.0]
-  def change
-    create_table :pictures do |t|
```

Comment:
Deleting this migration will cause the `DropPictures` migration to be invalid.

Reply:
Thanks for pointing that out. I was unsure how to go about this. I created the `droptable` migration, dropped the model and the controller, and then ran the migration. 

I think removing this migration may have been part of my git cleanup. 
I'll keep that in mind for future migrations. Thanks!